conductor-client/src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -731,6 +748,11 @@ class ExternalDatabaseManagementService(
         return "SELECT * FROM $tableName"
     }
 
+    private fun getExternalDatabaseTableSchemaSql(tableName: String): String {
+        return "SELECT ${PostgresColumn.PG_SCHEMA_NAME} FROM pg_catalog.pg_tables " +
+                "WHERE ${PostgresColumn.PG_TABLE_NAME} = '${tableName}'"
```

Comment:
this should be done as a PreparedStatement otherwise we're opening ourselves up to SQL injection

Reply:
Can you point me to an example? Should the other functions like this also be changed?

Reply:
You'd convert the `{tableName}` to `?` and then use a `PreparedStatementHolder` instead of `StatementHolder`